static/backbone/css/backgrid.css

```diff
@@ -285,3 +284,32 @@ td:hover .button-footer {
     border: none !important;
 }
 
+.backgrid th, .backgrid td {
+    padding: 0px !important;
```

Comment:
Not super important/a blocker, but if you have time, do you mind changing the !important stuff in a way that avoids !important - it can just be a pain later as !important rules accrue and usually can be avoided